README.md

```diff
@@ -0,0 +1,18 @@
+# DANDI Handbook
+Handbook for interacting with the DANDI archive.
+
+## DANDI Style Guidelines
+
+Follow the guidelines below when creating and revising text in the DANDI Handbook:
+
+*	**dandi-** repositories - hyphenate the names of DANDI GitHub repositories (e.g. **dandi-archive**); Dandisets is an exception because it is a
```

Comment:
```suggestion
*	**dandi-** repositories - Hyphenate the names of DANDI GitHub repositories (e.g. **dandi-archive**); "Dandisets" is an exception because it is a
```

Reply:
```suggestion
*	**dandi-** repositories - hyphenate the names of DANDI GitHub repositories (e.g. **dandi-archive**); "Dandisets" is an exception because it is a
```

Reply:
these are bulleted items not full sentences so I'm not capitalizing the first word, will add quotes to Dandisets though